recipes/vc-feature/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: vc-feature
+  version: {{ version }}
+
+build:
+  number: 0
+  track_features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
```

Comment:
See these are tied to Python version so that is how we are able to get this to work.